pkgs/top-level/all-packages.nix

```diff
@@ -21163,6 +21163,10 @@ in
     jre = openjdk11;
   };
 
+  prevo = callPackage ../applications/misc/prevo { };
+  prevo-data = callPackage ../applications/misc/prevo/data.nix { };
+  prevo-tools = callPackage ../applications/misc/prevo/tools.nix { };
```

Comment:
Why is prevo separate from prevo-tools?

Reply:
`prevo-tools` provides just the programs, without the data.

A user could want to use the `prevodb` executable to produce their own pre-processed data set, e.g. from a source data version not available in Nixpkgs. If so, they might not want to install the pre-processed data included in Nixpkgs.

A user could want to use the `prevo` executable to access pre-processed data they themselves produced as above, or that they got from a different source than Nixpkgs. If so, they might not want to install the pre-processed data included in Nixpkgs.

I therefore thought, it'd be a good idea to offer `prevo-tools` as a package of its own, providing only the programs without the data. Please let me know if this is at odds with Nixpkgs' intended structure.

Reply:
Thank you for making the `gpl2` &rarr; `gpl2Only` changes with https://github.com/NixOS/nixpkgs/compare/13566aa9b548aca1562b8fce7c7a112fb2c2e45b..905cf1555c59411672c7581109d0d7921e80c47e

I was still trying to find out whether my assessment of https://github.com/bpeel/revo 's licence was still correct.